site2/docs/functions-deploy-cluster-builtin.md

```diff
@@ -0,0 +1,16 @@
+---
+id: functions-deploy-cluster-builtin
```

Comment:
Suggest adding the link of this new topic to the [parent topic](https://pulsar.apache.org/docs/next/functions-deploy-cluster) to update the map for users.

Reply:
done